modules/server/src/utils/__tests__/dataToTSV.js

```diff
@@ -0,0 +1,262 @@
+import dataToTSVStream, { dataToTSV, columnsToHeader } from '../dataToTSV';
+import { PassThrough } from 'stream';
+
+describe('dataToTSV accessor columns', () => {
+  it('should handle string accessors', () => {
+    const config = {
+      index: 'file',
+      data: {
+        data: {
+          file: {
+            hits: {
+              edges: [
+                { node: { test1: 1, test2: 'txt1' } },
+                { node: { test1: 2, test2: 'txt2' } },
+              ],
+            },
+          },
+        },
+      },
+      columns: [
+        {
+          Header: 'Test1',
+          field: 'test1',
+          accessor: 'test1',
+        },
+        {
+          Header: 'Test2',
+          field: 'test2',
+          accessor: 'test2',
+        },
+      ],
+    };
+
+    const expected = 'Test1	Test2\n1	txt1\n2	txt2';
```

Comment:
can these be updated to show the tabs explicitly? `\t` this feels easily breakable in a way that might be hard to understand why

Reply:
done :)